modules/softwaremanagementplans/requirements.md

```diff
@@ -0,0 +1,77 @@
+---
+title: SMP requirements
+type: info
+order: 2
+---
+
+
+
+## **Purpose**
```

Comment:
yes, yes, no ;)

Reply:
If NEBULA does change that, would this need to be changed as well, to avoid some kind of double-bold state? Or will it then just be redundant that here it is indicated to be bold?